src/frontend/AppContainerWrapper.tsx

```diff
@@ -92,13 +99,42 @@ function hidePracticeModeTemporarily(route: string | null) {
   return route !== null && TEMP_HIDE_PRACTICE_MODE_UI.includes(route);
 }
 
+const AuthContainer = createAppContainer(AuthStack);
+
 const AppContainerWrapper = () => {
   const navRef = React.useRef<NavigationContainerComponent>();
   const [inviteModalEnabled, setInviteModalEnabled] = React.useState(true);
   const [queuedInvite, setQueuedInvite] = React.useState<string | null>(null);
   const [{ experiments }] = React.useContext(SettingsContext);
   const [hidePracticeBar, setHidePracticeBar] = React.useState(true);
   const [hidePracticeMode, setHidePracticeMode] = React.useState(false);
+  const [{ authStatus }, setAuthState] = React.useContext(SecurityContext);
+  const isMounted = React.useRef(false);
+
+  React.useEffect(() => {
+    isMounted.current = true;
+  }, []);
```

Comment:
this doesn't seem to be used? okay to remove? also fyi but we have a hook for this too: `useIsMounted`.

in the future, if this is just to prevent that common React warning about setting state on an unmounted component, this check will be completely unnecessary. there's an issue explaining how that warning is misleading and isn't actually a probably for unmounted components in most cases

Reply:
ahhh really? good to know!